parm/atms_beamwidth.txt

```diff
@@ -0,0 +1,36 @@
+# LOC = data/preproc/atms_beamwidth.dat
```

Comment:
Will this file change for different cycles?

Reply:
I have moved the atms_bean.txt from /parm to /fix/jedi directory as your suggestion.